hack/update-bindata.sh

```diff
@@ -37,14 +36,21 @@ if ! which go-bindata &>/dev/null ; then
 fi
 
 BINDATA_OUTPUT="${KUBE_ROOT}/test/e2e/generated/bindata.go"
-go-bindata -nometadata -prefix "${KUBE_ROOT}" -o ${BINDATA_OUTPUT} -pkg generated \
+go-bindata -nometadata -prefix "${KUBE_ROOT}" -o "${BINDATA_OUTPUT}.tmp" -pkg generated \
 	-ignore .jpg -ignore .png -ignore .md \
 	"${KUBE_ROOT}/examples/..." \
 	"${KUBE_ROOT}/docs/user-guide/..." \
 	"${KUBE_ROOT}/test/e2e/testing-manifests/..." \
 	"${KUBE_ROOT}/test/images/..." \
 	"${KUBE_ROOT}/test/fixtures/..."
 
-gofmt -s -w ${BINDATA_OUTPUT}
+gofmt -s -w "${BINDATA_OUTPUT}.tmp"
 
-V=2 kube::log::info "Generated bindata file : $(wc -l ${BINDATA_OUTPUT}) lines of lovely automated artifacts"
+if ! cmp  "${BINDATA_OUTPUT}.tmp" "${BINDATA_OUTPUT}" ; then
```

Comment:
compare and overwrite only if different to avoid always overwriting. 

Because it changes the file timestamps?

Why use `cat` and not `mv` or `cp`?